src/sentry/interfaces/http.py

```diff
@@ -64,7 +64,7 @@ def fix_broken_encoding(value):
 
 
 def jsonify(value):
-    return to_unicode(value) if isinstance(value, str) else json.dumps(value)
+    return to_unicode(value) if isinstance(value, str) else orjson.dumps(value).decode()
```

Comment:
I think this is leftover from python 2 -- can remove the `to_unicode` part of this at least since it does nothing